### PR TITLE
Make wp shell impossible in sub-shell mode

### PR DESCRIPTION
**Pull request by elazzabi**
This makes running `wp shell` impossible in the sub-shell mode

### CODE REVIEW

File: src/bin/vip-wp.js [hunk 1]
```diff
@@ -141,12 +141,18 @@ commandWrapper( {
 
 				const startsWithWp = line.startsWith( 'wp ' );
 				const empty = 0 === line.length;
+				const isShellCommand = 'wp shell' === line;
```
Comment (nickdaugherty) [inline src/bin/vip-wp.js]: I wonder if we should be more flexible in the check here, to handle any params (like `--url`) that get tacked on:
```suggestion
				const isShellCommand = line.startsWith( 'wp shell ' );
```

**Review by nickdaugherty — COMMENTED** [review]
What do you think about a more permissive check for `wp shell`?

**Comment by elazzabi** [comment]
@nickdaugherty you're right we need more permissive check, just committed 👍

**Review by nickdaugherty — APPROVED** [review]
🚢

**Comment by elazzabi** [comment]
one test is failing on Windows, can be fixed by relaunching the test, merging

**Pull request merged**